### PR TITLE
coucou

### PR DESCRIPTION
**Pull request by ServaneC**
added coucou file

### CODE REVIEW

File: coucou.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+coucou
```
Comment (ServaneC) [inline coucou.txt:1]: pas mal

**Review by estellesoulard — COMMENTED** [review]
non

File: coucou.txt (hunk 1 above)
Comment (estellesoulard) [reply coucou.txt:1]: this is a comment

**Review by estellesoulard — APPROVED** [review]
this is ok

**Pull request merged**